app/presenters/sufia/admin_set_presenter.rb

```diff
@@ -1,4 +1,7 @@
 module Sufia
   class AdminSetPresenter < CollectionPresenter
+    def total_items(admin_set = self)
```

Comment:
In a presenter the admin_set would be as `self.solr_document` (as you want the id you could just call `id`)